contracts/examples/digital-cash/wasm/src/lib.rs

```diff
@@ -20,13 +22,14 @@ multiversx_sc_wasm_adapter::endpoints! {
     digital_cash
     (
         init => init
+        claimFees => claim_fees
+        getAmount => get_amount
+        pay_fee_and_fund => pay_fee_and_fund
         fund => fund
+        depositFees => deposit_fees
         withdraw => withdraw
         claim => claim
```

Comment:
would like to see set/remove fee token. Also fee tokens should be a map of token/value. And SC owner can change those at any time.